ldms/src/core/ldms.c

```diff
@@ -100,9 +100,83 @@ const char *ldms_xprt_op_names[] = {
 	"DIR_REP",
 	"SEND",
 	"RECV",
+	"STREAM_PUBLISH",
+	"STREAM_SUBSCRIBE",
+	"STREAM_UNSUBSCRIBE"
 };
 static char *type_names[];
 
+/* -ENOSYS means that LDMS doesn't support profiling for those operation. */
+int enable_profiling[LDMS_XPRT_OP_COUNT] = {
+	0,          /* lookup */
+	0,          /* update */
+	-ENOSYS,    /* Publish */
+	0,          /* set_delete */
+	-ENOSYS,    /* dir_req */
+	-ENOSYS,    /* dir_rep */
+	0,          /* send */
+	-ENOSYS,    /* receive */
+	0,          /* stream_publish */
+	-ENOSYS,    /* stream_subscribe */
+	-ENOSYS	    /* stream_unsubscribe */
```

Comment:
Why are these error codes negative?

Reply:
I made the error code negative because I used 1 as enabled and 0 as disabled. I'll add a comment that includes the meanings of 0 and 1.